pingora-proxy/tests/utils/server_utils.rs

```diff
@@ -232,11 +232,9 @@ impl ProxyHttp for ExampleProxyHttp {
     ) -> Result<Box<HttpPeer>> {
         let req = session.req_header();
         if req.headers.contains_key("x-uds-peer") {
-            return Ok(Box::new(HttpPeer::new_uds(
-                "/tmp/nginx-test.sock",
-                false,
-                "".to_string(),
-            )));
+            return Ok(Box::new(
+                HttpPeer::new_uds("/tmp/nginx-test.sock", false, "".to_string()).unwrap(),
```

Comment:
With the change mentioned above, you should be able to change this to 
```suggestion
HttpPeer::new_uds("/tmp/nginx-test.sock", false, "".to_string())?,
```